Processes/How-To-Write-Content-for-the-docs-Portal.md

```diff
@@ -0,0 +1,12 @@
+## How To Write Content for the EOS Documentation Portal
```

Comment:
For consistency, the filename for this doc should have "Docs" in uppercase.

Reply:
Done